### DISCUSSION TITLE
Как вывести область / округ

### DISCUSSION THREAD

**Discussion by VychuzhaninovNA** [cdek-it/widget#53] [Q&A] [answered]
Добрый день.
При выборе ПВЗ в строку адреса у меня подставляется код отделения, город, улица и дом.
Например, VRT1 Воротынск, ул. 50 лет Победы, 16
Хотелось бы выводить еще и область.
Это возможно? В документации не нашел такого поля.

**Comment by vermorag** [comment, marked as answer]
Добрый день. В настоящий момент такого функционала нет.

**Reply by vermorag** [reply]
@VychuzhaninovNA Можете подсказать бизнес потребность такого действия?

**Reply by VychuzhaninovNA** [reply]
настроен расчет в модуле стороннего разработчика, но используется официальный api СДЭК
в модуле включено сопоставление регионов как подстраховка от одинаковых названий городов
иначе может подтянуться вообще другой город и список ПВЗ из другого города

кроме этого часто клиенты переживают, когда не видят региона в адресной строке, зная, что их город есть в разных регионах нашей страны.

**Reply by vermorag** [reply]
Это необходимо только для ПВЗ, или для выбранных адресов с карты тоже?

**Reply by VychuzhaninovNA** [reply]
только для ПВЗ, в настоящий момент до адреса наши клиенты выбрать не могут на карте.
в будущем может ситуация изменится.
если можно, до добавьте область во всех случаях выбора адреса на карте (ПВЗ или до домашнего адреса)
спасибо

**Reply by vermorag** [reply]
Если требуется, могу подключить вас к рассылке по релизу всех новых версий наших плагинов. Письма в рассылке о новых версиях содержат номера задач команды разработки, входящих в патч, а также краткое описание обновлений. Для удобства отслеживания вашего пожелание по доработке - номер задачи CMS-866. Срок подсказать не могу.
Для подписки на рассылку напишите, пожалуйста, в телеграм: https://t.me/cdek_it_bot , раздел Интеграция для интернет-магазинов.
